src/toil_scripts/adam_uberscript/adam_uberscript.py

```diff
@@ -388,7 +398,7 @@ def manage_standalone_spark_cluster(params, domain):
         semaphore = Semaphore.load(domain, name=standalone_spark_semaphore_name)
 
 
-def collect_realtime_metrics(params, threshold=0.5):
+def collect_realtime_metrics(params, threshold=0.025):
```

Comment:
Yeah, after reviewing the logs from my latest run, looks like 0.5% not 50%. I'll remove that change.